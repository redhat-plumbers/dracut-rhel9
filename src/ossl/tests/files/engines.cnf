openssl_conf = openssl_init

[openssl_init]
engines = engines_sect

[engines_sect]
afalg = afalg_sect
loader_attic = loader_attic_sect
pkcs11 = pkcs11_sect

[afalg_sect]
dynamic_path = afalg.so

[loader_attic_sect]
dynamic_path = /usr/lib64/engines-3/loader_attic.so
init = 1

[pkcs11_sect]
engine_id = pkcs11
dynamic_path = /usr/lib64/engines-3/libpkcs11.so
MODULE_PATH = opensc-pkcs11.so
init = 1
