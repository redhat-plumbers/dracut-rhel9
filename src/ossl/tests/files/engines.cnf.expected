/usr/lib64/engines-3/afalg.so
/usr/lib64/engines-3/loader_attic.so
/usr/lib64/engines-3/libpkcs11.so
/usr/lib64/ossl-modules/fips.so
